### PR TITLE
All the stuffings Dec 2018

### PR DESCRIPTION
**Pull request by RClement5700**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -51,17 +56,22 @@ dependencies {
...
     api 'com.unboundid:unboundid-ldapsdk:4.0.5'
+
     //login && openfire dependencies
     api 'org.igniterealtime.smack:smack-resolver-minidns:4.1.8'
     api 'org.igniterealtime.smack:smack-android:4.1.8'
     api 'org.igniterealtime.smack:smack-android-extensions:4.1.5'
     api 'org.igniterealtime.smack:smack-tcp:4.1.5'
     api 'org.igniterealtime.smack:smack-im:4.1.8'
+
+    //how to enable and disable 'offline mode'?
```
Comment (RClement5700) [inline app/build.gradle]: I added a comment!

**Comment by RClement5700** [comment]
@AlkalineGarrett take a look

File: app/src/main/java/com/catosystems/rohan/dynamicdelivery/activities/fragments/NotesFragment.java [hunk 2]
```diff
@@ -0,0 +1,110 @@
+//package com.catosystems.rohan.dynamicdelivery.activities.fragments;
```
Comment (AlkalineGarrett) [inline app/src/main/java/com/catosystems/rohan/dynamicdelivery/activities/fragments/NotesFragment.java]: So much dead code

File: app/src/main/java/com/catosystems/rohan/dynamicdelivery/activities/fragments/RegisterFragment.java [hunk 3]
```diff
@@ -61,154 +49,46 @@ public void replaceFragment(Fragment fragment) {
...
-                return headers;
-            }
-        };
-        System.err.println("Adding request to queue...");
-        queue.add(request);
-        System.err.println("Starting queue...");
-        queue.start();
     }
-
-    //    public void registerUser() {
```
Comment (AlkalineGarrett) [inline app/src/main/java/com/catosystems/rohan/dynamicdelivery/activities/fragments/RegisterFragment.java]: even more dead code!

**Pull request merged**